packages/venia-ui/lib/components/CartPage/PriceAdjustments/GiftOptions/giftOptions.js

```diff
@@ -0,0 +1,166 @@
+import React, { useMemo } from 'react';
+import gql from 'graphql-tag';
+
+import useGiftOptions from '@magento/peregrine/lib/talons/CartPage/GiftOptions/useGiftOptions';
+
+import Checkbox from '../../../Checkbox';
+import TextArea from '../../../TextArea';
+import { mergeClasses } from '../../../../classify';
+
+import defaultClasses from './giftOptions.css';
+
+const GiftOptions = props => {
+    const {
+        includeGiftReceipt,
+        includePrintedCard,
+        giftMessage,
+        toggleIncludeGiftReceiptFlag,
+        toggleIncludePrintedCardFlag,
+        updateGiftMessage
+    } = useGiftOptions({
+        getGiftOptionsQuery: GIFT_OPTIONS_QUERY,
+        saveGiftOptionsQuery: SET_GIFT_OPTIONS_QUERY
+    });
+
+    const classes = useMemo(() => mergeClasses(defaultClasses, props.classes), [
+        props.classes
+    ]);
```

Comment:
Don't bother memoizing this.

Reply:
Sure. Any particular reason?

Reply:
Just consistency. We don't memoize `mergeClasses()` in any of the existing components (afaik), and I have minor concerns about things slipping through the cracks when we get to refactoring `classify` one day.